CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.0.2
+  - Features Added
```

Comment:
does this release introduce new subgraph version ? 40_0 ?

Reply:
Yes, it should be included in the changelog?

Reply:
yes. along with arc.js new version
I think it is nice to have